WasatchNET/AndorEEPROM.cs

```diff
@@ -0,0 +1,137 @@
+﻿#if WIN32 || x64
```

Comment:
It's more because I don't really understand exactly how AnyCPU is supposed to work, and these libraries seem to have hard bit requirements, so I decided to just leave the feature out of that version.

These are just pre-processor directives so no runtime decisions and no guardrails.

Mac stuff is still living somewhere else